permissions/plugin-conjur-simple-integration.yml

```
@@ -0,0 +1,7 @@
+---
+name: "conjur-simple-integration"
+github: "jenkinsci/conjur-simple-integration"
+paths:
+- "org/conjur/jenkins/conjur-simple-integration"
```

```suggestion
- "io/jenkins/plugins/conjur-simple-integration"
```

This should be the groupId + artifactId in path form see your pom.xml: https://github.com/jenkinsci/conjur-simple-integration-plugin/blob/master/pom.xml#L10